<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(74, 62, 91, 255), stop:1 rgba(60, 63, 65, 255));
color: white</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(43, 45, 48);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>14</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>14</number>
        </property>
        <item>
         <widget class="QTabWidget" name="tabWidget_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>471</width>
            <height>281</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: black</string>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_19">
           <attribute name="title">
            <string>Низкоразрешенное изображение</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_43">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QLabel" name="label_24">
                <property name="styleSheet">
                 <string notr="true">color: white</string>
                </property>
                <property name="text">
                 <string>Визуализация открытого изображения c низким пространственным разрешением</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_20">
           <attribute name="title">
            <string>Высокроразрешенное изображение</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_44">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_45">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="graphicsView_26">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 31, 34);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_25">
                <property name="styleSheet">
                 <string notr="true">color: white</string>
                </property>
                <property name="text">
                 <string>Визуализация открытого изображения c высоким пространственным разрешением</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>471</width>
            <height>281</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">color: black</string>
          </property>
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_21">
           <attribute name="title">
            <string>Bicubic</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_46">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="graphicsView_27">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 31, 34);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_26">
                <property name="styleSheet">
                 <string notr="true">color: white</string>
                </property>
                <property name="text">
                 <string>Повышение пространственного разрешения методом бикубической интерполяции</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_22">
           <attribute name="title">
            <string>IHS</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_47">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_48">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="graphicsView_28">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 31, 34);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_27">
                <property name="styleSheet">
                 <string notr="true">color:white</string>
                </property>
                <property name="text">
                 <string>Повышение пространственного разрешения методом IHS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_23">
           <attribute name="title">
            <string>BT</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_49">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_50">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="graphicsView_29">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 31, 34);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_28">
                <property name="styleSheet">
                 <string notr="true">color: white</string>
                </property>
                <property name="text">
                 <string>Повышение пространственного разрешения методом BT</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_24">
           <attribute name="title">
            <string>GS</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_51">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_52">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="graphicsView_30">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 31, 34);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_29">
                <property name="styleSheet">
                 <string notr="true">color:white</string>
                </property>
                <property name="text">
                 <string>Повышение пространственного разрешения методом GS</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_25">
           <attribute name="title">
            <string>Wavelet</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_53">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_54">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="graphicsView_31">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 31, 34);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_30">
                <property name="styleSheet">
                 <string notr="true">color: white
</string>
                </property>
                <property name="text">
                 <string>Повышение пространственного разрешения методом Wavelet</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_26">
           <attribute name="title">
            <string>Super Resolution</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_55">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_56">
              <property name="spacing">
               <number>5</number>
              </property>
              <item>
               <widget class="QGraphicsView" name="graphicsView_32">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(30, 31, 34);</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_31">
                <property name="styleSheet">
                 <string notr="true">color: white</string>
                </property>
                <property name="text">
                 <string>Повышение пространственного разрешения методом Super Resolution</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>14</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white</string>
          </property>
          <property name="title">
           <string>Выбор методов:</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QCheckBox" name="checkBox">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>Bicubic</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_2">
             <property name="text">
              <string>IHS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_3">
             <property name="text">
              <string>GS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_4">
             <property name="text">
              <string>Brovey T</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_5">
             <property name="text">
              <string>Wavelet</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBox_6">
             <property name="text">
              <string>Super Resolution</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>PushButton</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_3">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>935</width>
             <height>365</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(30, 31, 34);
color: white</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1097</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Открыть изображение с низким пространственным разрешением</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Открыть изображение с высоким пространственным разрешением</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
